src/zope/configuration/tests/test_docs.py

```diff
@@ -49,12 +48,15 @@
 )
 
 def test_suite():
+    suite = unittest.TestSuite()
     here = os.path.dirname(os.path.abspath(__file__))
     while not os.path.exists(os.path.join(here, 'setup.py')):
         prev, here = here, os.path.dirname(here)
         if here == prev:
             # Let's avoid infinite loops at root
-            raise AssertionError('could not find my setup.py')
+            print('tests_doc.py: WARNING could not find setup.py -->'
```

Comment:
I wonder: does raising `SkipTest` do anything useful here?

If not, what about returning a suite with one test case that *does* raise `SkipTest`? That way at least it's something the test runner can report about, and can go in XML files, etc.

Reply:
Raising `SkipTest` directly leads to `module could not be imported` which leads to a non-zero exit code.